src/main.ts

```diff
@@ -42,23 +42,49 @@ app.get('/.well-known/server-health', (req, res) => {
   res.status(200).send('ok');
 });
 
-// The parameters we expect end users to provide
-interface BrazePocketHitsQuery {
-  scheduledSurfaceID: string;
+// The url params
+enum BrazePocketHitsParams {
+  scheduledSurfaceID = 'scheduledSurfaceID',
+}
+
+// The query parameters we expect end users to provide
+interface BrazePocketHitsQueryParams {
   date: string;
 }
 
-app.get('/scheduled-items/:scheduledSurfaceID?date=:date', async (req, res) => {
-  // enable 30 minute cache when in AWS
-  if (config.app.environment !== 'development') {
-    res.set('Cache-control', 'public, max-age=1800');
+// A custom request type to help with code completion
+type BrazePocketHitsRequest = Request<
+  Record<BrazePocketHitsParams, 'string'>,
+  any,
+  any,
+  BrazePocketHitsQueryParams
+>;
+
+// base path after our domain uri
+const basePath = '/scheduled-items/';
+
+/**
+ * Controller function for the path: /scheduled-items/:scheduledSurfaceID'
+ */
+app.get(
+  `${basePath}:${BrazePocketHitsParams.scheduledSurfaceID}`,
```

Comment:
`${basePath}:${BrazePocketHitsParams.scheduledSurfaceID}` I know this seems like over kill so I'm happy to get rid of this 😄 

We don't really have any other endpoints right now so it makes sense just hard coding it